refarch-backend/src/main/java/de/muenchen/refarch/configuration/nfcconverter/NfcRequest.java

```diff
@@ -196,7 +197,7 @@ public ServletInputStream getInputStream() throws IOException {
 
         log.debug("Converting InputStream data to NFC.");
         final String nfcConvertedContent = NfcHelper.nfcConverter(content);
-        return new NfcServletInputStream(new ByteArrayInputStream(nfcConvertedContent.getBytes()));
+        return new NfcServletInputStream(new ByteArrayInputStream(nfcConvertedContent.getBytes(StandardCharsets.UTF_8)));
```

Comment:
Are you sure that's what needs to be done here? I am not certain about the use of this class. Should the identified encoding in line `190` be used instead?

Reply:
i think for that usecase its better to not specify a charset. So i removed it again and added a suppression to the function

Reply:
That's fine. :)